src/components/organisms/OnBoarding/components/Modal.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import styled from 'styled-components';
+import CloseButton from '../../Notification/NotificationHeader/CloseButton';
+import { PopinClose } from 'components/molecules/Popin/Popin';
+import { CloseFunction } from '../index';
+import useNoScrollBody from '../../../../app/utils/useNoScrollBody';
```

Comment:
Lot's of `..`.
I suppose we could use an `effects` now that we have a few.

Reply:
i didn't get it, sorry ?

Reply:
Put all effects `useNoScrollBody` `useThis` `useThat` into an effect `directory`

Reply:
I mean if there are more than 1

Reply:
ah okay